src/core/library.c

```diff
@@ -1893,7 +1875,7 @@ QuicTraceRundown(
             LibraryRundown,
             "[ lib] Rundown, PartitionCount=%u DatapathFeatures=%u",
             MsQuicLib.PartitionCount,
-            CxPlatDataPathGetSupportedFeatures(MsQuicLib.Datapath));
+            MsQuicLib.Datapath != NULL ? CxPlatDataPathGetSupportedFeatures(MsQuicLib.Datapath) : 0);
```

Comment:
Let's split this event. The way you have it, you can't tell from the rundown if it's initialized and supports no features or if it's not initialized.